package zephyr.matching;

enum AssertionType
{
  EQUALS = 0;
  NOT = 1;
  GREATER_THAN = 2;
  GREATER_THAN_OR_EQUAL = 3;
  LESS_THAN = 4;
  LESS_THAN_OR_EQUAL = 5;
  ANY = 6;
};